marlowe-playground-client/src/Simulation.purs

```diff
@@ -327,6 +321,24 @@ handleGistAction settings LoadGist = do
 
   toEither x NotAsked = x
 
+loadGist ::
+  forall m.
+  MonadAff m =>
+  MonadEffect m =>
+  Gist ->
+  HalogenM State Action ChildSlots Void m (Either String Unit)
```

Comment:
I don't know if you mind, but I think the inferred type signature would be more abstract.

Reply:
inferred type is the same